packages/tasks/src/account.ts

```
@@ -43,39 +46,105 @@ export const getWalletFromRecoveryInput = async (ethers: any) => {
+        message: "Solana private key (press Enter to skip):",
+      });
+      if (solanaPrivateKey.trim() === "") {
+        console.log("Skipped Solana private key input.");
+        return null;
+      }
+      try {
+        return Keypair.fromSecretKey(bs58.decode(solanaPrivateKey));
+      } catch (e) {
+        console.error(`❌ Invalid Solana private key: ${e}`);
+        continue;
+      }
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)